tests/features/asset_distribution/add_asset_distribution.feature

```diff
@@ -16,9 +16,10 @@ Feature: Add asset distribution through the UI
       | title         | Asset release random name        |
       | description   | Some reusable random description |
       | is version of | Asset random name                |
-
     When I am logged in as a "moderator"
     And I go to the homepage of the "Asset release random name" asset release
+
+    Then I click "Add"
```

Comment:
Maybe this should be "Then I click the + sign" or 'the plus icon' or something similar. There is no 'Add' in the page.